### ISSUE TITLE
The character '@' is a reserved JQL character

### ISSUE THREAD

**Issue by TMiguelT** [ralphbean/bugwarrior#644]
Based on the discussion [here](https://community.atlassian.com/t5/Jira-questions/JIRA-Rest-API-authentication-always-returns-401-unauthorized/qaq-p/187181), it seems that JIRA cloud usernames are now your login email address. What this means is that bugwarrior is generating JQL queries that contain the `@` symbol, and thus fails. The error log I'm getting in this situation is this (I've removed lines that mention my email address and login token, in case you're wondering why some lines are missing):
```
INFO:bugwarrior.db:Service-defined UDAs exist: you can optionally use the `bugwarrior-uda` command to export a list of UDAs you can add to your taskrc file.
INFO:bugwarrior.services:Starting to aggregate remote issues.
INFO:bugwarrior.services:Spawning 1 workers.
INFO:bugwarrior.services:Working on [genovic_jira]
	text: Error in the JQL Query: The character '@' is a reserved JQL character. You must enclose it in a string or use the escape '\u0040' instead. (line 1, character 24)
	response text = {"errorMessages":["Error in the JQL Query: The character '@' is a reserved JQL character. You must enclose it in a string or use the escape '\\u0040' instead. (line 1, character 24)"],"errors":{}}
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/bugwarrior/services/__init__.py", line 494, in _aggregate_issues
    for issue in service.issues():
  File "/usr/local/lib/python3.6/dist-packages/bugwarrior/services/jira.py", line 310, in issues
    cases = self.jira.search_issues(self.query, maxResults=-1)
  File "/usr/local/lib/python3.6/dist-packages/jira/client.py", line 2109, in search_issues
    issues = self._fetch_pages(Issue, 'issues', 'search', startAt, maxResults, search_params)
  File "/usr/local/lib/python3.6/dist-packages/jira/client.py", line 575, in _fetch_pages
    resource = self._get_json(request_path, params=page_params, base=base)
  File "/usr/local/lib/python3.6/dist-packages/jira/client.py", line 2549, in _get_json
    r = self._session.get(url, params=params)
  File "/usr/local/lib/python3.6/dist-packages/jira/resilientsession.py", line 151, in get
    return self.__verb('GET', url, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/jira/resilientsession.py", line 147, in __verb
    raise_on_error(response, verb=verb, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/jira/resilientsession.py", line 57, in raise_on_error
    r.status_code, error, r.url, request=request, response=r, **kwargs)
```

**Comment by ralphbean** [comment]
Note to self: #652 did not resolve this.

**Comment by my-flow** [comment]
+1

**Comment by baartosz** [comment]
workaround: setting custom query for gathering tickets seems to avoid the problem:
`jira.query = assignee = ralph and status != closed and status != resolved`
https://bugwarrior.readthedocs.io/en/latest/services/jira.html#specify-the-query-to-use-for-gathering-issues

**Comment by haakobja** [comment]
The workaround that @baartosz listed works as expected. If your username is an email address `assignee` can be set as either `user\u0040domain.tld` or `'user@domain.tld'`. 

So maybe the `assignee` part of the default `jira.query` should/could be quoted as default?

**Comment by ryneeverett** [comment]
Resolved via #753.
